tests/cut-plane/test_detection.py

```diff
@@ -80,9 +80,17 @@ def test_minimize_more17():
     params = 4, 45, -21, 0, 0, 61
 
     result = _minimize(params, _get_points(), bin_width=10)
-    assert_allclose(result,
-                    [4.1510e+00, 4.6763e+01, -2.0966e+01, -2.7207e-04, -4.5070e-04, 6.8804e+01],
-                    rtol=1e-4)
+    if (
+        version.parse(np.__version__) < version.parse('1.25')
+        or "AVX512_SKX" not in np.show_config(mode="dicts").get("SIMD Extensions", {}).get("found", {})
+    ):
```

Comment:
@mgeplf is this ok for you?

Reply:
> It's not clean but I found a way :)

Cool and scary at the same time :)

> @mgeplf is this ok for you?

Yeah, let's get this unblocked.  If we have a chance to find a better method, that would be great.